src/support/colors.h

```diff
@@ -20,6 +20,8 @@
 #include <iosfwd>
 
 namespace Colors {
+void setEnabled(bool enabled);
+bool isEnabled();
 void disable();
```

Comment:
I think we should remove the `disable()` API, and replace all callers with calls to `setEnabled(false)`.